src/blockchain/deposit_strategy/curated_module.py

```diff
@@ -43,6 +49,11 @@ def is_deposited_keys_amount_ok(self) -> bool:
 
     def _get_possible_deposits_amount(self) -> int:
         depositable_ether = self.w3.lido.lido.get_depositable_ether()
+        if (is_mellow_depositable(self.w3, self.module_id) and
```

Comment:
Move
`self.w3.lido.lido.get_buffered_ether() >= self.w3.lido.lido_locator.withdrawal_queue_contract.unfinalized_st_eth())` to is_mellow_depositable method

Reply:
Done